robottelo/orm.py

```diff
@@ -20,7 +20,7 @@
 #: Default for ``poll_rate`` argument to :func:`robottelo.orm._poll_task`.
 TASK_POLL_RATE = 5
 #: Default for ``timeout`` argument to :func:`robottelo.orm._poll_task`.
-TASK_TIMEOUT = 180
+TASK_TIMEOUT = 300
```

Comment:
We can flip it back to 180, if there is no improvement seen in the automation results.